chef/cookbooks/ceph/templates/default/ceph.conf.erb

```diff
@@ -141,3 +141,10 @@
   <% end -%>
 <% end -%>
 <% end -%>
+
+<% if (! node['ceph']['config_sections'].nil?) -%>
```

Comment:
This the value is not nil but a empty hash, so I guess that this will be always `true`?

Maybe `if !node['ceph']['config_sections'].empty?` is more correct